src/main/react/app/containers/App/index.js

```diff
@@ -0,0 +1,210 @@
+/**
+ *
+ * App
+ *
+ * This component is the skeleton around the actual pages, and should only
+ * contain code that should be seen on all pages. (e.g. navigation bar)
+ */
+
+import React from "react";
+
+import "react-datepicker/dist/react-datepicker.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./style/index.css";
+import ExcludedDate from "./excludedDate";
+import TimeRange from "./timeRange";
+import { getExcludedDates, setExcludedDates } from "../../api";
+import lodash from 'lodash'
```

Comment:
I seem to recall lodash is pretty bloated. I think some of the features you're using here are available from other packages. Maybe there's webpack magic to make it smaller though?

Reply:
Nope for this import way, but if we use 
`
import {cloneDeep} from 'lodash'
`
Then webpack is powerful enough to only import and integrate the cloneDeep function into the final output.
I'll optimize my code.